src/app/_components/FilterContainer.tsx

```diff
@@ -15,7 +15,7 @@ type FilterProps = {
 
 type ResultsProps = {
   category: ReactNode
-  results: ReactNode
+  results?: ReactNode
```

Comment:
How come this is optional now?

Reply:
This is because we render `ResultsAndReset` for the ecosystem projects, which is pass via this prop.

For blogs and events, we don't a global filter reset anymore.